YoCode/Results.cs

```diff
@@ -63,13 +67,13 @@ public void ApplySpecialCases(List<FeatureEvidence> list)
 
         public void AssignToEquivalentCheck(FeatureEvidence oldCheck,FeatureEvidence newCheck)
         {
+            newCheck.FeatureWeighting = oldCheck.FeatureWeighting;
```

Comment:
Is this needed? I feel like `newCheck.FeatureWeighting` wouldn't be used unless oldCheck is inconclusive, so this line would not be necessary. Am I right?